dlvm/tools/gpu-burn/LICENSE

```diff
@@ -0,0 +1,25 @@
+BSD 2-Clause License
```

Comment:
are we forking someone else's code and put it here?  please check and confirm that this is okay.